docs/overview/core-concepts-office-add-ins.md

```diff
@@ -9,77 +8,13 @@ localization_priority: Priority
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="../images/index-landing-page/i_code-blocks.svg" alt="Develop" />
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Develop</h3>
-                        <p><a href="../develop/develop-overview.md">Learn important development concepts for Office Add-ins.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-    <li>
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="../images/index-landing-page/i_design.svg" alt="Design" />
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>UI and UX</h3>
-                        <p><a href="../design/interface-elements.md">Learn about designing the user interface and user experience for Office Add-ins.</p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-    <li>
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="../images/index-landing-page/i_recommended-testing.svg" alt="Testing and debugging" />
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Test and debug</h3>
-                        <p><a href="../testing/test-debug-office-add-ins.md">Learn how to test and debug Office Add-ins.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-    <li>
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="../images/index-landing-page/i_deploy.svg" alt="Publishing" />
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Publish</h3>
-                        <p><a href="../publish/publish.md">Learn how to deploy and publish Office Add-ins.</a></p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-</ul>
\ No newline at end of file
+|               |               |
+| ------------- | ------------- |
+| ![code-blocks](../images/index-landing-page/i_code-blocks.svg)  | **Develop** [Learn important development concepts for Office Add-ins.](../develop/develop-overview.md) |
```

Comment:
The icons are too small. Can you make them larger?
![image](https://user-images.githubusercontent.com/9088056/84058366-00044d00-a96e-11ea-8604-64cc1bcf8b23.png)